GoogleTestAdapter/Packaging.TAfGT/Packaging.TAfGT.csproj

```diff
@@ -26,7 +26,7 @@
     <CopyOutputSymbolsToOutputDirectory>true</CopyOutputSymbolsToOutputDirectory>
     <BypassVsixValidation>true</BypassVsixValidation>
     <DeployExtension Condition="'$(TestAdapterFlavor)' != 'TAfGT'">false</DeployExtension>
-    <IsProductComponent>true</IsProductComponent>
+    <IsProductComponent>$(IsProductComponent)</IsProductComponent>
```

Comment:
Wouldn't it be more canonical to do Condition and set true/false accordingly?

Reply:
This is interesting since it's using the replacement from Microbuild for this actually. I do agree we should set it default here in case this isn't set (for local build for instance).